content/lang/tr/guide/switching-to-preact.md

```diff
@@ -3,40 +3,38 @@ name: Switching to Preact from React
 permalink: '/guide/switching-to-preact
 ---
 
-# Switching to Preact (from React)
+# React'tan Preact'a geçiş
 
-There are two different approaches to switch from React to Preact:
+Preact'tan React'a geçişte iki türlü yaklaşım vardır:
 
-1. Install the `preact-compat` alias
-2. Switch your imports to `preact` and remove incompatible code
+1. `preact-compat` alias ı yüklemek
+2. import ettiğiniz dosyaları `preact` a çevirmek ve uyumsuz kodları silmek
 
-## Easy: `preact-compat` Alias
+## Kolay olanı: `preact-compat`
 
-Switching to Preact can be as easy as installing and aliasing `preact-compat` in for `react` and `react-dom`.
-This lets you continue writing React/ReactDOM code without any changes to your workflow or codebase.
-`preact-compat` adds somewhere around 2kb to your bundle size, but has the advantage of supporting
-the vast majority of existing React modules you might find on npm.  The `preact-compat` package provides
-all the necessary tweaks on top of Preact's core to make it work just like `react` and `react-dom`, in a single module.
+Preact'a geçmek `preact-compat` i `react` ve `react-dom` için kısayol (alias) olarak göstermek kadar kolaydır.
+Bu sizin çalışma şeklinizde ve ya projenizdeki kodlarda hiçbir değişiklik yapmadan React/ReactDOM kodu yazmaya devam etmenize olanak tanır.
+`preact-compat` projenizde yaklaşık olarak 2 kb yük getirirken aynı zamanda ihtiyacınız olan tüm desteği sağlar.
+npm üzerinde  React modüllerinin büyük çoğunluğunun desteği `preact-compat` tarafından sağlanmaktadır.
+Tek bir modülün içerisinde Preact'ı `react` ve `react-dom` gibi çalışmasına yetecek şekilde ihtiyacınız olan tüm ayarlar...
```

Comment:
The `preact-compat` package provides all the necessary tweaks on top of Preact's core to make it work just like `react` and `react-dom`, in a single module.

`preact-compat` paketi, Preact'in çekirdeğinin tek bir modülde `react` and `react-dom` gibi çalışmasını sağlayacak değişiklikleri sağlar.